### PR TITLE
Rebuild packages for missing metadata

### PR DESCRIPTION
**Pull request by Chocimier**
This is a proposition to rebuild packages for complement missing metadata in *-repodata indexes in order to make them suitable to use by Repology, so Void Linux could be finally added there.

All packages was build on x86_64, x86_64-musl, armv7hf and aarch64-musl and passed check stage on x86_64 and x86_64-musl. Nothing was tested manually whether it still works.

564 more packages remain to rebuild.

[ci skip]

### CODE REVIEW

**Comment by maxice8** [comment]
Mind taking some time to fix up some templates that are ugly ?

**Comment by Chocimier** [comment]
To be honest, I didn't even read them. What's wrong?

File: srcpkgs/dvdauthor/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'dvdauthor'
 pkgname="dvdauthor"
```
Comment (maxice8) [inline srcpkgs/dvdauthor/template]: doesn't need to be quoted

File: srcpkgs/dvdauthor/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 version="0.7.2"
```
Comment (maxice8) [inline srcpkgs/dvdauthor/template]: ditto

File: srcpkgs/fbxkb/template [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 pkgname="fbxkb"
```
Comment (maxice8) [inline srcpkgs/fbxkb/template]: unquote here as well and remove the empty line above

File: srcpkgs/kbd/template [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'kbd'
 pkgname=kbd
```
Comment (maxice8) [inline srcpkgs/kbd/template]: license is GPL-2.0-or-later

File: srcpkgs/libdiscid/template [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libdiscid'
 pkgname=libdiscid
 version=0.6.2
```
Comment (maxice8) [inline srcpkgs/libdiscid/template]: license is LGPL-2.1-or-later

Comment (maxice8) [reply srcpkgs/libdiscid/template]: and the -devel package short desc should have only one dash

File: srcpkgs/m4/template [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'm4'
 pkgname=m4
 version=1.4.18
```
Comment (maxice8) [inline srcpkgs/m4/template]: license is GPL-3.0-or-later

File: srcpkgs/mdadm/template [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'mdadm'
 pkgname=mdadm
 version=4.0
```
Comment (maxice8) [inline srcpkgs/mdadm/template]: maintainer should be

Leah Neukirchen <leah@vuxu.org>

license is GPL-2.0-or-later

File: srcpkgs/mktorrent/template [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'mktorrent'
 pkgname=mktorrent
 version=1.1
```
Comment (maxice8) [inline srcpkgs/mktorrent/template]: license is GPL-2.0-or-later

**Comment by Chocimier** [comment]
Additional comment when you see no more things to change would be useful.

**Comment by maxice8** [comment]
On Tue, Oct 23, 2018 at 04:26:45PM +0000, Piotr wrote:
> Additional comment when you see no more things to change would be useful.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4086#issuecomment-432317176

?

File: srcpkgs/atomicparsley/template [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'atomicparsley'.
 pkgname=atomicparsley
 version=0.9.0
```
Comment (maxice8) [inline srcpkgs/atomicparsley/template]: GPL-2.0-or-later

File: srcpkgs/dosfstools/template [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'dosfstools'
 pkgname=dosfstools
 version=4.1
```
Comment (maxice8) [inline srcpkgs/dosfstools/template]: GPL-3.0-or-later

Comment (maxice8) [reply srcpkgs/dosfstools/template]: and license= should be between maintainer= and homepage=

File: srcpkgs/ed/template [hunk 11]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'ed'
 pkgname=ed
 version=1.14.2
```
Comment (maxice8) [inline srcpkgs/ed/template]: GPL-3.0-or-later

File: srcpkgs/libgadu/template [hunk 12]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'libgadu'
 pkgname=libgadu
 version=1.12.2
```
Comment (maxice8) [inline srcpkgs/libgadu/template]: LGPL-2.1-only

File: srcpkgs/lxappearance/template [hunk 13]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'lxappearance'
 pkgname=lxappearance
 version=0.6.3
```
Comment (maxice8) [inline srcpkgs/lxappearance/template]: GPL-2.0-or-later

File: srcpkgs/lxpanel/template [hunk 14]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'lxpanel'
 pkgname=lxpanel
 version=0.9.3
```
Comment (maxice8) [inline srcpkgs/lxpanel/template]: GPL-2.0-or-later

File: srcpkgs/lxtask/template [hunk 15]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'lxtask'
 pkgname=lxtask
 version=0.1.8
```
Comment (maxice8) [inline srcpkgs/lxtask/template]: GPL-2.0-or-later

File: srcpkgs/nkf/template [hunk 16]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'nkf'
 pkgname=nkf
 version=2.1.4
```
Comment (maxice8) [inline srcpkgs/nkf/template]: Zlib

File: srcpkgs/parcellite/template [hunk 17]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'parcellite'
 pkgname=parcellite
 version=1.2.1
```
Comment (maxice8) [inline srcpkgs/parcellite/template]: GPL-3.0-or-later

**Comment by Chocimier** [comment]
> ?
I mean: please let me know when you check all files, so I can fix them all at once just after that.

File: srcpkgs/perl-LWP-Protocol-https/template [hunk 18]
```diff
@@ -1,7 +1,7 @@
 # Template build file for 'perl-LWP-Protocol-https'.
 pkgname=perl-LWP-Protocol-https
 version=6.07
```
Comment (maxice8) [inline srcpkgs/perl-LWP-Protocol-https/template]: Artistic-1.0-Perl, GPL-1.0-or-later

Leah Neukirchen <leah@vuxu.org>

File: srcpkgs/perl-Log-Log4perl/template [hunk 19]
```diff
@@ -1,7 +1,7 @@
 # Template build file for 'perl-Log-Log4perl'
 pkgname=perl-Log-Log4perl
 version=1.49
```
Comment (maxice8) [inline srcpkgs/perl-Log-Log4perl/template]: Artistic-1.0-Perl, GPL-1.0-or-later

File: srcpkgs/polkit-qt/template [hunk 20]
```diff
@@ -10,7 +10,7 @@ maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="https://projects.kde.org/projects/kdesupport/polkit-qt-1"
 license="LGPL-2.1"
```
Comment (maxice8) [inline srcpkgs/polkit-qt/template]: LGPL-2.1-or-later

File: srcpkgs/puddletag/template [hunk 21]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'puddletag'
 pkgname=puddletag
 version=1.2.0
```
Comment (maxice8) [inline srcpkgs/puddletag/template]: GPL-3.0-or-later

File: srcpkgs/python-pyx/template [hunk 22]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'python-pyx'
 pkgname=python-pyx
 version=0.12.1
```
Comment (maxice8) [inline srcpkgs/python-pyx/template]: GPL-2.0-or-later

File: srcpkgs/python-urlgrabber/template [hunk 23]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'python-urlgrabber'
 pkgname=python-urlgrabber
 version=3.10.2
```
Comment (maxice8) [inline srcpkgs/python-urlgrabber/template]: LGPL-2.1-or-later

**Comment by maxice8** [comment]
On Tue, Oct 23, 2018 at 09:54:03AM -0700, Piotr wrote:
> > ?
> 
> I mean: please let me know when you check all files, so I can fix them all at once just after that.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/4086#issuecomment-432328719

Ah sure, i am working on them

File: srcpkgs/rdfind/template [hunk 24]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'rdfind'
 pkgname=rdfind
 version=1.3.5
```
Comment (maxice8) [inline srcpkgs/rdfind/template]: GPL-2.0-or-later

File: srcpkgs/run-mailcap/template [hunk 25]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'run-mailcap'
 pkgname="run-mailcap"
```
Comment (maxice8) [inline srcpkgs/run-mailcap/template]: remove quotes around version and pkgname

File: srcpkgs/swig/template [hunk 26]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'swig'
 pkgname=swig
 version=3.0.12
```
Comment (maxice8) [inline srcpkgs/swig/template]: GPL-3.0-or-later

File: srcpkgs/xfce4-systemload-plugin/template [hunk 27]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'xfce4-systemload-plugin'
 pkgname=xfce4-systemload-plugin
 version=1.2.1
```
Comment (maxice8) [inline srcpkgs/xfce4-systemload-plugin/template]: BSD-2-Clause

File: srcpkgs/xfce4-netload-plugin/template [hunk 28]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'xfce4-netload-plugin'
 pkgname=xfce4-netload-plugin
 version=1.3.1
```
Comment (maxice8) [inline srcpkgs/xfce4-netload-plugin/template]: GPL-2.0-or-later

File: srcpkgs/xiccd/template [hunk 29]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'xiccd'
 pkgname=xiccd
 version=0.2.4
```
Comment (maxice8) [inline srcpkgs/xiccd/template]: GPL-3.0-or-later

File: srcpkgs/xtrlock/template [hunk 30]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'xtrlock'
 pkgname=xtrlock
 version=3.4
```
Comment (maxice8) [inline srcpkgs/xtrlock/template]: GPL-2.0-or-later

File: srcpkgs/ykneomgr/template [hunk 31]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'ykneomgr'
 pkgname=ykneomgr
 version=0.1.8
```
Comment (maxice8) [inline srcpkgs/ykneomgr/template]: LGPL-3.0-or-later

**Comment by Chocimier** [comment]
Templates changed. BTW git rebase --autosquash is cute.

**Comment by maxice8** [comment]
mdadm was updated no need to rebuild

**Comment by Chocimier** [comment]
> mdadm was updated no need to rebuild

Dropped.

**Pull request merged**